src/include/rho/FixedVector.hpp

```diff
@@ -222,9 +222,8 @@ namespace rho {
 	// Virtual function of GCNode:
 	void detachReferents() override;
     private:
-	T* m_data;  // pointer to the vector's data block.
```

Comment:
@kmillar after running the benchmark I couldn't see any change in execution time from this commit.